CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Screen share zoom
 - Localization support 
 - Update joining experience in setup view
+- Localization support 
 
```

Comment:
Could you delete the second Localization support?
There is one before Update joining experience in setup view.